packages/sheets-formula-ui/src/views/range-selector/hooks/useHighlight.ts

```diff
@@ -48,14 +49,12 @@ export function useSheetHighlight(isNeed: boolean, unitId: string, subUnitId: st
     const refSelectionsRenderService = render?.with(RefSelectionsRenderService);
     const sheetSkeletonManagerService = render?.with(SheetSkeletonManagerService);
 
-    const [ranges, rangesSet] = useState<ISelectionWithStyle[]>([]);
-
-    useEffect(() => {
+    const highlightSheet = (refSelections: IRefSelection[]) => {
         const workbook = univerInstanceService.getUnit<Workbook>(unitId);
         const worksheet = workbook?.getActiveSheet();
         const selectionWithStyle: ISelectionWithStyle[] = [];
```

Comment:
is there a way to known who triggered selection change?  by input on the right or pointer events in the spreadsheet?